### DISCUSSION TITLE
cmdx not installed by registry set up

### DISCUSSION THREAD

**Discussion by scop** [aquaproj/aqua#3287] [Deprecated (Support Request)] [answered]
### aqua info

```console
$ aqua info
{
  "version": "",
  "commit_hash": "",
  "os": "linux",
  "arch": "amd64",
  "pwd": "/home/(USER)/Documents/code/aqua-registry",
  "root_dir": "/home/(USER)/.local/share/aquaproj-aqua",
  "env": {},
  "config_files": [
    {
      "path": "/home/(USER)/Documents/code/aqua-registry/aqua.yaml"
    }
  ]
}
```

### Overview

https://aquaproj.github.io/docs/products/aqua-registry/contributing/#cmdx---task-runner says "cmdx is installed by [Set up](https://aquaproj.github.io/docs/products/aqua-registry/contributing/#set-up) already"

I just tried to set up for the first time according to https://aquaproj.github.io/docs/products/aqua-registry/contributing/#set-up, and cmdx is not installed for me with "aqua i -l". (Well at least it's not in $PATH.)

### How to reproduce

aqua.yaml

```yaml

```

Other related code such as local Registry

```

```

Executed command and output

```console
$ 
```

### Debug output

```console
$ 
```

### Expected behaviour

cmdx installed in $PATH like the instructions indicate.

### Actual behaviour

$ cmdx
cmdx: command not found

**Comment by suzuki-shunsuke** [comment]
It looks weird.
cmdx should be installed by `aqua i -l`.

https://github.com/aquaproj/aqua-registry/blob/541aa1f65f931f28fb8ff4ef7a247364718d40c8/aqua.yaml#L13

Seems like you build aqua yourself.

> ```
> {
>   "version": "",
>   "commit_hash": "",
> ```

How did you install aqua?
Could you try to install aqua using [aqua-installer](https://aquaproj.github.io/docs/products/aqua-installer#shell-script)?

**Reply by scop** [reply]
I installed it with `go install`. I prefer not to run (or bother even reviewing) arbitrary installer scripts if there are options available.

**Comment by suzuki-shunsuke** [comment, marked as answer]
Do you set the environment variable `$PATH`?

https://aquaproj.github.io/docs/install/#2-set-the-environment-variable-path

```sh
export PATH="${AQUA_ROOT_DIR:-${XDG_DATA_HOME:-$HOME/.local/share}/aquaproj-aqua}/bin:$PATH"
```

**Reply by scop** [reply]
Ah, indeed, I missed this step -- it is so far down on the install page that I did not notice that there are more than one step in the first place, and stopped reading when I found my preferred install method. `cmdx` is there, cheers.
